tests-arquillian/src/test/java/org/jboss/weld/tests/ejb/mdb/EJBTest.java

```diff
@@ -51,20 +48,16 @@
                 .addAsModule(
                         ShrinkWrap.create(BeanArchive.class, Utils.getDeploymentNameAsHash(EJBTest.class))
                                 .addPackage(EJBTest.class.getPackage())
-                        //.addAsManifestResource(EJBTest.class.getPackage(), "test-destinations-service.xml", "test-destinations-service.xml")
                 );
     }
 
-    @Category(Broken.class)
     @Test
-    @Ignore
-    // TODO Need a way to deploy test-destinations-service.xml to JBoss AS
     public void testMdbUsable(Control control) throws Exception {
         InitialContext ctx = new InitialContext();
         QueueConnectionFactory factory = (QueueConnectionFactory) ctx.lookup("ConnectionFactory");
         QueueConnection connection = factory.createQueueConnection();
         QueueSession session = connection.createQueueSession(false, Session.AUTO_ACKNOWLEDGE);
-        Queue queue = (Queue) ctx.lookup("queue/testQueue");
+        Queue queue = (Queue) ctx.lookup("java:jboss/exported/jms/queue/test");
```

Comment:
Well I think yes because https://github.com/weld/core/blob/master/tests-arquillian/src/test/java/org/jboss/weld/tests/util/WeldExtension.java handles this in https://github.com/weld/core/blob/master/tests-arquillian/src/test/java/org/jboss/weld/tests/util/WildFly8EEResourceManager.java#L40

We can export this string as a constant e.g in https://github.com/weld/core/blob/master/tests-arquillian/src/test/java/org/jboss/weld/tests/util/WildFlyMessaging.java